### PR TITLE
Adding Shraddha to hindi directory.

### PR DESCRIPTION
**Pull request by neerajmohinani**
Adding to hindi dir. must be automatically merged.

### CODE REVIEW

File: hindi/shraddha/shraddha.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Shraddha Kapoor (born 3 March 1989) is an Indian film actress and singer who works in Bollywood films.
```
Comment (casey-collab) [inline hindi/shraddha/shraddha.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by pmallory** [comment]
@casey-collab Can you review this PR please?

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

**Comment by neerajmohinani** [comment]
@casey-collab can you merge it now?

**Comment by casey-collab** [comment]
Looks great! :heavy_check_mark:

**Pull request merged**